website/src/content/help-needed.mdx

```diff
@@ -46,3 +46,76 @@ This implies that changes to the [Custom Application configuration](https://docs
 If you are interested in more advanced functionalities, let us know and open a [support issue](https://github.com/commercetools/merchant-center-application-kit/issues/new/choose).
 
 </Info>
+
+# I want to use a test or staging environment for my Custom Application
+
+During the development cycle of new features for a Custom Application, it can be helpful to showcase or test those new features before publishing them to the production environment.
+
+This generally can be achieved using preview deployments, where you can inspect different versions of the Custom Application. As of now, the Merchant Center does not support this feature yet.
+
+The only option to emulate this setup is to create multiple Custom Applications in the Merchant Center, each one with a different [`entryPointUriPath`](/api-reference/application-config#entrypointuripath).
+
+However, the Custom Application must be configured to support "dynamic" values for the `entryPointUriPath` and build different versions of it.
+
+<Info>
+
+Custom Applications "dynamic" configurations (per environment) setup are further explained in [this section](/api-reference/application-config#environment-variable-references).
+
+</Info>
+
+The very same Custom Application can be deployed several times and set its [configuration file](/api-reference/application-config) to use different `entryPointUriPath` values by taking advantage of environment variables for every deployment.
+
+Let's say there are two deployments of the same application: `A` and `B`. We can assign an environment value for the `entryPointUriPath` property in its configuration.
+
+```json
+{
+  "entryPointUriPath": "${env:ENTRY_POINT_URI_PATH}"
+}
+```
+
+The `ENTRY_POINT_URI_PATH` environment variable can be provided in different ways:
+
+For example:
+- as an inline environment variable when running a script command
+
+  ```console
+  ENTRY_POINT_URI_PATH=my-application-entry-point mc-scripts start
+  ```
+
+- using a [dotenv](https://github.com/motdotla/dotenv) file
+
+  ```bash title=".env-A"
+  ENTRY_POINT_URI_PATH=my-application-entry-point
+  ```
+
+  ```console
+  mc-scripts --env .env-A start
+  ```
+
+- by defining the environment variables in your CI service
+
+Using this pattern, different `entryPointUriPath` values can be defined for different environments:
+* `A` environment: `my-application-entry-point`
+* `B` environment: `my-application-entry-point-test`
+
+With this setup, a new preview version of the Custom Application can be deployed to `B` environment and showcasing/testing can be done without affecting the production environment.
```

Comment:
```suggestion
With this setup, preview versions of the Custom Application can be deployed to the `B` environment and tested using the application `my-application-entry-point-test`.
```

Reply:
Updated in this commit: 7bdca6330f1f6e6bc799395e5524ae1f4a2b7f9d